src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/MetaKeywords.java

```diff
@@ -78,7 +78,7 @@ public List<String> getMetaKeywords(TypeElement typeElement) {
             results.addAll(getMemberKeywords(utils.getFields(typeElement)));
             results.addAll(getMemberKeywords(utils.getMethods(typeElement)));
         }
-        ((ArrayList)results).trimToSize();
+        results.trimToSize();
```

Comment:
I wonder if this method will look better this way:
```
    public List<String> getMetaKeywords(TypeElement typeElement) {
        var results = new ArrayList<String>();
        ...
        results.trimToSize();
        return results;
    }
```

Reply:
Agreed; I was just trying to limit the cleanup. `var` is good.